components/pipeline-service/production/stone-prd-m01/resources/kustomization.yaml

```diff
@@ -21,3 +21,12 @@ patches:
       group: external-secrets.io
       version: v1beta1
       kind: ExternalSecret
+  - target:
+      kind: ExternalSecret
+      name: tekton-chains-public-key
+    patch: |
+      $patch: delete
```

Comment:
I'm unfamiliar if whether gitops delete complains enough if the entry is not there, and the gitops rollout is prevented / stays out of sync.

Do you have experience with this @enarha or somehow was able to confirm this via some form of local testing?

Maybe @hugares knows how gitops will react?

I suppose the alternative, which I suspect you were trying to avoid @enarha, is duplicating the `components/base/external-secrets/openshift-pipelines` entries in `staging/base` and then `production/base`

Reply:
kustomize does not complain if the target is not there. And yes, that I found the easier alternative than duplicating configuration and making the next change (enable prod) easier as well. Once the change is merged, it's easy to verify and we can quickly proceed with updating prod, effectively removing that bit in a day or two.

Reply:
cool deal